config_template.cfg

```diff
@@ -753,6 +753,10 @@ MARKER_ANALYZE_AVERAGE = MASSFLUX
 % Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
 NUM_METHOD_GRAD= GREEN_GAUSS
 %
+% Numerical method for spatial gradients used for upwind reconstruction
+% (NONE, GREEN_GAUSS, LEAST_SQUARES, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD_RECON= LEAST_SQUARES
+%
```

Comment:
That is material for the docs page IMO

Reply:
Jury is still out whether it will necessarily be less accurate.. it will be very case-dependent. 

Nevertheless, I will be adding some text to the docs about tuning the solver. Essentially the same content that I have written in the PR description above.